controllers/feeds/getFeedsV2.js

```diff
@@ -110,11 +111,20 @@ module.exports = async (req, res) => {
 
   try {
     const {token} = req;
-    const myAnonymousUser = await UsersFunction.findAnonymousUserId(User, req.userId, {raw: true});
-    // START get excluded post parameter
-    const listBlockUser = await getListBlockUser(req.userId);
-    const listBlockDomain = await getBlockDomain(req.userId);
-    const listPostAnonymousAuthor = await getListBlockPostAnonymousAuthor(req.userId);
+    const [
+      myAnonymousUser,
+      // START get excluded post parameter
+      listBlockUser,
+      listBlockDomain,
+      listPostAnonymousAuthor,
+      isBlurredPost
+    ] = await Promise.all([
+      UsersFunction.findAnonymousUserId(User, req.userId, {raw: true}),
+      getListBlockUser(req.userId),
+      getBlockDomain(req.userId),
+      getListBlockPostAnonymousAuthor(req.userId),
+      UserFollowUserFunction.checkIsBlurredPost(UserFollowUser, User, req.userId)
+    ]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `Promise.all` here is a good improvement for performance as it allows all the promises to be executed concurrently. However, it's important to note that if any promise rejects, `Promise.all` will immediately reject with the reason of the first promise that rejected, which might not be what you want if you wish to continue execution even if one of the promises fails. If you want to handle each error individually and continue execution regardless, consider using `Promise.allSettled` instead.



<!-- This is an auto-generated comment by OSS CodeRabbit -->